lib/membrane_ice_plugin/ice_endpoint.ex

```diff
@@ -507,10 +474,7 @@ defmodule Membrane.ICE.Endpoint do
   end
 
   @impl true
-  def handle_info({:DOWN, _ref, _process, alloc_pid, _reason}, _ctx, state) do
-    alloc_span_id(alloc_pid)
-    |> Membrane.OpenTelemetry.end_span()
-
+  def handle_info({:DOWN, _ref, _process, _alloc_pid, _reason}, _ctx, state) do
```

Comment:
If this callback does nothing, there is no reason to call `Process.monitor/1` in line 467